app/models/user.rb

```diff
@@ -115,6 +116,26 @@ def hybrid?
     school_id && responsible_body_id
   end
 
+  # Wrapper methods to ease the transition from 'user belongs_to school',
+  # to 'user has_many schools'
+  def school
+    schools.first
+  end
+
+  def school_id
+    school&.id
+  end
+
+  def school_id=(new_school_id)
+    user_schools.delete_all
+    schools << School.find(new_school_id) if new_school_id
```

Comment:
i feel this guard should also guard the line above? otherwise a mistake might delete all the associations

Reply:
I found a few cases in the specs where we do things like `user.update!(school_id: nil)` to remove a user from a school - this construction is to support that case.